### DISCUSSION TITLE
Highlight some 3D building with different color

### DISCUSSION THREAD

**Discussion by GreenLightSM** [rnmapbox/maps#1312] [Q&A] [answered]
Hi! I have 3D buildings on map, but I have some list with coordinates and I want to fill buildings that match this coordinates with different color. I'm using react-native and https://github.com/react-native-mapbox-gl/maps
I want to do something like this:
![image](https://user-images.githubusercontent.com/34767722/114751548-f30a7400-9d5d-11eb-93fb-5eee2cf5cfe5.png)

**Comment by ferdicus** [comment, marked as answer]
https://github.com/react-native-mapbox-gl/maps/blob/master/docs/FillExtrusionLayer.md#name-3

**Reply by GreenLightSM** [reply]
I use this to highlight all buildings, but how can I highlight some buildings in different color?
